lib/tpm2_errata.c

```diff
@@ -143,6 +143,9 @@ void tpm2_errata_fixup(tpm2_errata_index_t index, ...) {
         return;
     }
 
+    LOG_WARN("Applying errata: \"%s\", use --disable-errata to disable!",
+            errata->name);
```

Comment:
If we flip the logic to instead have an option to enable erratas, I think we should change this to LOG_INFO().

Reply:
Agreed.

Reply:
There is an Errata applied statement at the end...